apps/keyboard/js/imes/emoji/emoji.js

```diff
@@ -0,0 +1,121 @@
+/* global InputMethods, PromiseStorage*/
+
+
+(function () {
+  'use strict';
```

Comment:
Move `'use strict';` to line 3 please.

Let's create a boring ES5 constructor and do `InputMethods.emoji = new EmojiInputMethod();` at the very end.

Please remove all ES6 since we would like to have http://timdream.org/gaia-keyboard-demo/ continue to work on iOS Safari.